packages/xchain-thorchain/src/client.ts

```diff
@@ -613,6 +631,32 @@ class Client extends BaseXChainClient implements ThorchainClient, XChainClient {
       return getDefaultFees()
     }
   }
+
+  private async determineGasLmit({
+    txBody,
+    privKey,
+    multiplier,
+    fallbackGasLimit,
+  }: {
+    txBody: proto.cosmos.tx.v1beta1.TxBody
+    privKey: proto.cosmos.crypto.secp256k1.PrivKey
+    multiplier: number
+    fallbackGasLimit: string
+  }): Promise<string> {
+    try {
+      const gas_expected = await getGasExpectedForTx({ cosmosSDKClient: this.getCosmosClient(), txBody, privKey })
+      if (gas_expected === undefined) {
+        throw new Error('could not estimate gas limit required for TX')
+      }
+      const gasLimit = cosmosclient.Long.fromString(gas_expected).multiply(multiplier).toString()
+
+      return gasLimit
+    } catch (error) {
+      console.error(error)
+      console.warn(`Using the fallbackGasLimit of ${fallbackGasLimit} `)
+      return fallbackGasLimit
```

Comment:
Don't return a hard-coded fallback. If `getGasExpectedForTx` fails there is a reason and broadcasting a tx won't work after that. Because `getGasExpectedForTx` includes a call of `buildUnsignedTx` which should not fail.

Reply:
hmm interesting...i was thinking if there is some kind of issue with calling /simulate, the tx should at least give it a try   it another try with the back up gas limit? I'll let you respond before removing the  fallback entirely